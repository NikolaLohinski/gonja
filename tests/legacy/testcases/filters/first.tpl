{{ "Test"|first }}
{{ (complex.comments | first).Text  }}
{{ 5|first }}
{{ true|first }}
{{ nothing|first }}
{{ simple.chinese_hello_world|first }}
