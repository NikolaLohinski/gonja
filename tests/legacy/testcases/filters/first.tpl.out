T
"gonja is nice!"



ä½ 
